src/api/java/cvc5/Solver.java

```diff
@@ -26,36 +27,2502 @@ public long getPointer()
+   * \endverbatim
+   * @param symbol the name of the function
+   * @param sorts the sorts of the parameters to this function
+   * @param sort the sort of the return value of this function
+   * @return the function
+   */
+  public Term declareFun(String symbol, Sort[] sorts, Sort sort)
+  {
+    long[] sortPointers = Utils.getPointers(sorts);
+    long termPointer = declareFun(pointer, symbol, sortPointers, sort.getPointer());
+    return new Term(this, termPointer);
+  }
+
+  private native long declareFun(
+      long pointer, String symbol, long[] sortPointers, long sortPointer);
+
+  /**
+   * Declare uninterpreted sort.
+   * SMT-LIB:
+   * \verbatim
+   *   ( declare-sort <symbol> <numeral> )
+   * \endverbatim
+   * @param symbol the name of the sort
+   * @param arity the arity of the sort
+   * @return the sort
+   */
+  public Sort declareSort(String symbol, int arity) throws CVC5ApiException
+  {
+    Utils.validateUnsigned(arity, "arity");
+    long sortPointer = declareSort(pointer, symbol, arity);
+    return new Sort(this, sortPointer);
+  }
+
+  private native long declareSort(long pointer, String symbol, int arity);
+
+  /**
+   * Define n-ary function in the current context.
+   * SMT-LIB:
+   * \verbatim
+   *   ( define-fun <function_def> )
+   * \endverbatim
+   * @param symbol the name of the function
+   * @param bound_vars the parameters to this function
+   * @param sort the sort of the return value of this function
+   * @param term the function body
+   * @return the function
+   */
+  public Term defineFun(String symbol, Term[] bound_vars, Sort sort, Term term)
+  {
+    return defineFun(symbol, bound_vars, sort, term, false);
+  }
+
+  /**
+   * Define n-ary function.
+   * SMT-LIB:
+   * \verbatim
+   *   ( define-fun <function_def> )
+   * \endverbatim
+   * @param symbol the name of the function
+   * @param bound_vars the parameters to this function
+   * @param sort the sort of the return value of this function
+   * @param term the function body
+   * @param global determines whether this definition is global (i.e. persists
+   *               when popping the context)
+   * @return the function
+   */
+  public Term defineFun(String symbol, Term[] bound_vars, Sort sort, Term term, boolean global)
+  {
+    long[] boundVarPointers = Utils.getPointers(bound_vars);
+    long termPointer =
+        defineFun(pointer, symbol, boundVarPointers, sort.getPointer(), term.getPointer(), global);
+    return new Term(this, termPointer);
+  }
+
+  private native long defineFun(long pointer,
+      String symbol,
+      long[] boundVarPointers,
+      long sortPointer,
+      long termPointer,
+      boolean global);
+
+  /**
+   * Define n-ary function in the current context.
+   * SMT-LIB:
+   * \verbatim
+   * ( define-fun <function_def> )
+   * \endverbatim
+   * Create parameter 'fun' with mkConst().
+   * @param fun the sorted function
+   * @param bound_vars the parameters to this function
```

Comment:
```suggestion
   * @param boundVars the parameters to this function
```

Reply:
This applies to all instances of `bound_vars`.

Reply:
Done